### PR TITLE
Add IsDataAccessInCardDesignerEnabled flag

### PR DESCRIPTION
**Pull request by nick-pape**
See documentation here: https://learn.microsoft.com/en-us/sharepoint/dev/spfx/viva/features/card-designer/card-designer-api-support

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -2779,6 +2780,29 @@ Accept pipeline input: False
...
+### -IsDataAccessInCardDesignerEnabled 
+
+The IsDataAccessInCardDesignerEnabled settings (defaulted to false) will allow Viva Connections Dashboard operators to access SharePoint and Graph API's in the Card Designer card.
+
+The valid values are:  
+
+- False (default) – SharePoint and Graph API's cannot be accessed in the Card Designer card.
+- True – Users with edit permissions on the Dashboard will be able to access SharePoint and Graph API's in the Card Designer card. 
+
+Note: Further documentation on this feature is available [here](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/viva/features/card-designer/card-designer-api-support).
```
Comment (JohanFreelancer9) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2792]:
```suggestion
For further documentation on this feature, see [Overview of Viva Connections Card Designer advance API features](https://learn.microsoft.com/en-us/sharepoint/dev/spfx/viva/features/card-designer/card-designer-api-support).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @nick-pape, kindly commit the suggested change. Many thanks.

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -2779,6 +2780,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IsDataAccessInCardDesignerEnabled 
+
+The IsDataAccessInCardDesignerEnabled settings (defaulted to false) will allow Viva Connections Dashboard operators to access SharePoint and Graph API's in the Card Designer card.
+
+The valid values are:
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2787]: >   [](http://example.com/codeflow?start=21&length=2)

remove trailing spaces

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -2779,6 +2780,29 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -IsDataAccessInCardDesignerEnabled 
+
+The IsDataAccessInCardDesignerEnabled settings (defaulted to false) will allow Viva Connections Dashboard operators to access SharePoint and Graph API's in the Card Designer card.
+
+The valid values are:  
+
+- False (default) – SharePoint and Graph API's cannot be accessed in the Card Designer card.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2789]: >API's [](http://example.com/codeflow?start=41&length=5)

should be "APIs" not "API's"

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -2779,6 +2780,29 @@ Accept pipeline input: False
...
+- True – Users with edit permissions on the Dashboard will be able to access SharePoint and Graph API's in the Card Designer card.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:2790]: >API's [](http://example.com/codeflow?start=98&length=5)

should be "APIs" not "API's"

**Comment by nick-pape** [comment]
@samkabue @JohanFreelancer9 should be good to go now!

**Review by samkabue — APPROVED** [review]
:shipit:

**Pull request merged**